tests/RazorLight.WebTests/RazorLight.WebTests.csproj

```diff
@@ -0,0 +1,70 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
```

Comment:
unless this project is a shell to be ran by an `[WebApplicationFactory](https://docs.microsoft.com/en-us/aspnet/core/test/integration-tests?view=aspnetcore-5.0)` i would recommend moving back to standard SDK, and add a `FrameworkReference` into that project.

My (possibly wrong) assymption here is that the test project is relying on the `FrameworkReference` that the `Microsoft.NET.Sdk.Web` does for consumer. `FrameworkReference` is a better approach is all you need is the `Reference` out of it